themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,30 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group'] && strtolower($this->config->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'): ?> 
+                    <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+		    <?php $authorCount = count($authors) ?>
+		    <?php $authorNameLimit = $this->config->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                    <?php if (!empty($authors)): ?>
+                        <b><?=$this->transEsc($item['Label'])?>:</b>
+                        <?php foreach ($authors as $i => $author): ?>
+                            <a href="<?=$this->record($driver)->getLink('author', $this->highlight($author, null, true, false))?>"><?=$this->highlight($author)?></a>
+                            <?php if ($authorCount == ($i + 1)): ?>
+                                <?php break; ?>
+                            <?php elseif (($i + 1) > $authorNameLimit): ?>
+                                <?=$this->transEsc('more_authors_abbrev')?>
+			        <?php break; ?>
+                            <?php else:?>
+                                <?php echo $this->transEsc(','); ?>
```

Comment:
You don't need to `transEsc` the comma -- there is no such translation string. In theory, it might be good to add a translation string named something like `author_list_separator` so this could be more easily translated, but that's probably a separate issue/project. For now I'd just leave this as a plain comma to prevent confusion.

Reply:
thanks @demiankatz I've added that in my latest follow-up!

Reply:
Do you need to change it back to `<?php echo ','; ?>` to fix the whitespace issues we previously discussed, or is that no longer an issue now?